cmd/monaco/integrationtest/v2/bucket_integration_test.go

```diff
@@ -0,0 +1,52 @@
+//go:build integration
+// +build integration
+
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
+	"github.com/spf13/afero"
+)
+
+// tests all configs for a single environment
+func TestIntegrationBucket(t *testing.T) {
+
+	configFolder := "test-resources/integration-bucket/"
+	manifest := configFolder + "manifest.yaml"
+	specificEnvironment := ""
+	t.Setenv("MONACO_FEAT_BUCKETS", "1")
```

Comment:
I think the actual feature flag is still missing!

Reply:
Nevermind, I forgot that https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1124 already introduced the flag and loading bucket configs is behind the flag.